docs/.vuepress/site-config/sidebar-nav.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
         "policies/traffic-permissions",
         "policies/traffic-route",
         "policies/traffic-metrics",
-        "policies/traffic-tracing",
```

Comment:
Won't this change break links to `TrafficTrace` policy on all versions prior to `0.3.2` ?

Reply:
You're right, thanks